src/gyro/Gyro.ts

```diff
@@ -0,0 +1,136 @@
+import Chuck from "../Chuck";
+import { Gyro_ck, GyroMsg_ck } from "./gyroCk";
+
+
+/**
+ * Introducing Gyro (gyroerometer, on mobile) support for WebChucK. Gyro wraps
+ * JavaScript DeviceMotionEvent listeners easing access to mobile device gyroerometers
+ * in WebChucK code. 
+ *
+ * To get started with Gyro:
+ * @example
+ * ```ts
+ * import { Chuck, Gyro } from "webchuck";
+ *
+ * const theChuck = await Chuck.init([]);
+ * const gyro = await Gyro.init(theChuck); // Initialize Gyro
+ * ```
+ * 
+ * The "deviceorientation" event gives motion of the device around the three axes (x, y, and z) represented as degrees from 0 to 360.
+ * More on the "deviceorientation" event can be found online {@link https://developer.mozilla.org/en-US/docs/Web/API/Window/deviceorientation_event | here }.
+ * 
+ * iOS devices require that the web developer ask permission from the user to access sensors after a button push. This looks like: 
+ * 
+ * ```ts
+ * import { Chuck, Gyro} from '../webchuck/src/wc-bundle.js';
```

Comment:
When mentioning importing for documentation, you can just say `import { Chuck, Gyro } from "webchuck"`. The location is different on every platform so it's easier to keep it general.

Nit: Can you make sure there's a opening and closing space between `{ Chuck, Gyro }`